### DISCUSSION TITLE
How to jump to other apps?

### DISCUSSION THREAD

**Discussion by nyssance** [tauri-apps/tauri#10919] [Q&A]
Such as obsidian://xxx.md，in chrome, it's jump default, but tauri on work.

Work on windows, but not on macOS.

**Comment by HuakunShen** [comment]
Are you trying to open other apps from a tauri app?

I believe you can run a shell command. On windows use `start obsidian://`
https://v2.tauri.app/plugin/deep-linking/#windows

**Reply by nyssance** [reply]
On Windows, jump to other app with link such obsidian://xxx work fine. not need use command,
but not work on macOS.
